Readme.md

```diff
@@ -225,7 +233,7 @@ $ git submodule update --init --recursive
 
 ## TODOs
 
-At the moment Only UIView specific properties are supported. In the future we're going to expand Configurator to work on all UIKit View Subclasses (like UILabel e.g.).
+At the moment Only UIView specific properties are supported. In the future we're going to expand Configurator to work on all UIKit View Subclasses (like UIPageControl, UIScrollView e.g.).
 Also we want to provide some convenice configurations, like using UIColor for CGColor configurations, a shadow configuration set and extensions for third party libraries like ReactiveCocoa.
 
```

Comment:
We should also add mentions of the issue and the other library where we got the help we needed